docs/product-assembly_business-optimizer/src/main/asciidoc/main.adoc

```diff
@@ -16,8 +17,11 @@ Use this document to set up and configure {PLANNER} functionality.
 The *Employee Rostering* sample project has been created in {CENTRAL} to demonstrate Business Optimizer capabilities. To view this project, you must have {PRODUCT} installed. For more information about setting up and installing {PRODUCT} and {CENTRAL}, see _Getting started with decision services_.
 
 // Modules
-
+//About
 include::product-business-resource-planner-guide/optimizer-about-business-optimizer-con.adoc[leveloffset=+1]
+include::product-business-resource-planner-guide/optimizer-planning-problem-con.adoc[leveloffset=+1]
+
+//Downloading and installing
 include::product-business-resource-planner-guide/optimizer-download-proc.adoc[leveloffset=+1]
```

Comment:
Again, since the content file is not included, just a note that I _think_ we're going with the passive on prerequisites. Makes me crazy and I much prefer the active, like you have it (Download and install ...), but passive is the decided style I think for prereqs because it leans toward the thing already being done, and not always by the other in some cases. If it's _not_ likely to already have been done (which would call for active), then the argument is that it shouldn't be a prereq, but part of the procedure. That's the logic for the decision anyway. So the prereqs here, e.g., would be "{PRODUCT} is (or has been) downloaded and installed from..." Emily and I were just talking about this the other day. The choice is yours, but just a heads-up on the likely direction we're going. Emily should be clarifying sometime down the road.

Only mentioning this here, btw, just as a general note.

Reply:
Interesting. I tend to not bother getting involved in these debates because they take up too much time and I can't engage in real time with everyone else, so I'll change it here anyway since it sounds like you and Emily are already doing it.